radish34/zkp/src/utils/jsonifyVk.js

```diff
@@ -50,7 +50,6 @@ const keyExtractor = solFilePath => {
  */
 export const jsonifyVk = async solFilePath => {
   const vkJson = await keyExtractor(solFilePath);
-  // console.log('vkJson:', vkJson);
```

Comment:
`logger.debug` here instead of removing.

Reply:
Done.